tests/main/security-udev-input-subsystem/task.yaml

```diff
@@ -23,6 +23,16 @@ restore: |
     rm -f call.error
 
 execute: |
+    if [ ! -d /dev/input/by-path/ ]; then
```

Comment:
Can `systems` be updated now?

Reply:
I agree with Maciej, let's remove `systems` stanza above.